src/Microsoft.Dnx.Runtime/ApplicationHostContext.cs

```diff
@@ -13,11 +13,15 @@ public class ApplicationHostContext
     {
         public Project Project { get; set; }
 
+        public LockFile LockFile { get; set; }
```

Comment:
I like this change. It'll  allow us to remove the stupid skip lock file validation flag passed into here and it'll also help the caching situation in DTH.